packages/graphql/schemas/schema.graphql

```diff
@@ -2380,6 +2380,7 @@ type Studio {
 }
 
 enum StudioStatusType {
```

Comment:
What's the flow is statuses now? How are `ENABLED` and `INITIALIZED` used?

Reply:
`INITIALIZED` happens first. It means that the cloud studio bundle has been fetched and studio is available for use. `ENABLED` means that the correct environment variables were passed and protocol is enabled for studio (i.e. the cloud studio button and panel will appear)